docs/source/share_kedro_viz.md

```diff
@@ -11,6 +11,16 @@ The publish and share feature on Kedro-Viz enables seamless sharing of pipeline
 
 3. **Publish and Share Kedro-Viz with Github Actions**: Use the `publish-kedro-viz` action available on the Github Marketplace to deploy Kedro-Viz for your Kedro project repository on GitHub Pages. For further information and usage instructions, please refer to [publish-kedro-viz](https://github.com/marketplace/actions/publish-kedro-viz)
 
+## Filtering and sharing Kedro-Viz pipelines 
+
+Kedro-Viz allows user to focus on specific aspects of their data pipeline by filtering the visualization. Once filters are applied, the resulting visualization can be shared with others by sharing the URL
```

Comment:
```suggestion
Kedro-Viz allows you to focus on specific aspects of your data pipeline by filtering the visualization. Once filters are applied, you can share the resulting visualisation with others by copying the URL
```

?

Reply:
lol i had it as you before and then i changed the tone to 'users' :D

Reply:
I usually follow https://ddbeck.com/second-person-is-ok/ and https://github.com/kedro-org/kedro/wiki/Kedro-documentation-style-guide#style